.version

```diff
@@ -1 +1 @@
-v2.1
+v2.2.0
```

Comment:
Should just be v2.2, the release script in the Sansible repo controls the patch level.